webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -39,6 +39,8 @@ def setUp(self):
         super(TestViews, self).setUp()
         self.tmp_dir = tempfile.mkdtemp()
 
+        settings.SYMBOLS_MIME_OVERRIDES['jpeg'] = 'text/plain'
```

Comment:
NB: this is something you'll have to do in the tests themselves. Not in the `setUp`.